index.src.html

```diff
@@ -284,6 +284,30 @@ <h3 dfn export id="referrer-policy-no-referrer-when-downgrade" oldids="referrer-
 
   This is a user agent's default behavior, if no policy is otherwise specified.
 
+  <h3 dfn export id="referrer-policy-same-origin">"<code>same-origin</code>"</h3>
+
+  The <a>"<code>same-origin</code>"</a> policy specifies that a
+  full URL, <a href="#strip-url">stripped for use as a referrer</a>, is sent as
+  referrer information when making <a>same-origin requests</a> from a particular
+  <a>request client</a>.
+
+  <a>Cross-origin requests</a>, on the other hand, will contain no
+  referrer information. A <code><a>Referer</a></code> HTTP header will not be
+  sent.
+
+
+  <div class="example">
+    If a document at <code>https://example.com/page.html</code> sets a policy of
+    <a>"<code>same-origin</code>"</a>, then navigations to any
```

Comment:
Indeed, it was a copy paste error from the [origin-when-cross-origin example](https://github.com/w3c/webappsec-referrer-policy/blob/cedd5ee4202192ae312883713aec2f60f49d613b/index.src.html#L334). I repushed my branch and fixed the original typo in #51.